### DISCUSSION TITLE
Error on trying to render initial index.qmd: "ERROR: Unsupported project type manuscript"

### DISCUSSION THREAD

**Discussion by DCEW** [quarto-dev/quarto-cli#9628] [Q&A] [answered]
### Description

I've just forked then cloned the quarto-cli repo to try to render a manuscript template. I'm using RStudio on a corporate Windows laptop.  I've made no changes to `index.qmd` but when I click the render button, the error "ERROR: Unsupported project type manuscript" shows up in the Background Jobs window and the manuscript is not rendered - am I doing something silly? TIA!

**Comment by mcanouil** [comment]
You need to upgrade Quarto CLI to a version that has Manuscript project type, at least 1.3.

https://quarto.org/docs/get-started/

**Reply by DCEW** [reply]
I've got Quarto CLI version 1.3.450 - surely that's ok?

**Reply by mcanouil** [reply]
Are you sure about that?
What happen if you run `quarto check` inside the project where you have your Manuscript setup?
Do you run into the error, when using `quarto render` inside the same project from a terminal?

**Reply by DCEW** [reply]
This is the relevant part of `quarto check` in the directory of the cloned repo: 

```
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
Sending fatal alert BadCertificate
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.450
      Path: C:\Users\clwe3\AppData\Local\Programs\Quarto\bin
      CodePage: 1252
```

**Reply by mcanouil** [reply]
Please use proper formatting such as code block to share block of code.

**Reply by DCEW** [reply]
And I am unable to use the terminal in RStudio due to IT permissions, but running `quarto render` in the CMD in this same directory leads to the same error as I posted above: `ERROR: Unsupported project type manuscript

Error: Unsupported project type manuscript
    at projectType (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:58189:15)
    at projectContext (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:82774:30)
    at async render (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:90657:21)
    at async Command.fn (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:90856:32)
    at async Command.execute (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:8437:13)
    at async quarto (file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:127545:5)
    at async file:///C:/Users/clwe3/AppData/Local/Programs/Quarto/bin/quarto.js:127563:9`

**Reply by mcanouil** [reply]
~~Did you try to render from the terminal and not using your IDE interface?~~

By the way, you might also want to upgrade your RStudio IDE here as the issue might be the IDE rather than Quarto CLI.

**Reply by DCEW** [reply]
I've requested an upgrade of RStudio and the Quarto CLI from our IT dept. Presumably this wouldn't fix it though since it gives the same error in the CMD?

**Reply by mcanouil** [reply, marked as answer]
Ok, I realised that Manuscript project is a 1.4 feature <https://quarto.org/docs/manuscripts/>.

So, you don't have it unless you upgrade Quarto CLI (or RStudio, which embeds Quarto CLI).

**Reply by DCEW** [reply]
Ah ok, thanks so much, I'll get on to IT about it,  thanks for your time :-)
